python/tchannel/tornado/message_factory.py

```diff
@@ -60,7 +63,11 @@ def build_raw_request_message(self, request, args, is_completed=False):
             message = CallRequestMessage(
                 flags=request.flags,
                 # ttl=request.ttl,
-                # tracing=request.tracing,
+                tracing=Tracing(request.tracing.span_id,
+                                request.tracing.parent_span_id,
+                                request.tracing.trace_id,
+                                request.tracing.traceflags
+                                ),
```

Comment:
Move `)` to previous line. Usually, `)` goes on a separate line only if the `(` was on a separate line too. Like so,

    Tracing(
        arg1,
        arg2,
    )